### PR TITLE
What's new in SeaQuery 0.28.0

### PR DESCRIPTION
**Pull request by billy1624**

### CODE REVIEW

**Review by billy1624 — COMMENTED** [review]
Hey @ikrivosheev, please check :)

**Review by ikrivosheev — APPROVED** [review]
@billy1624 thank you! LGTM!

File: Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    .build(sea_query::MysqlQueryBuilder);
+
+// old behavior
+assert_eq!(statement, r#"SELECT `id` FROM `glyph`"#);
+
+// new behavior
+assert_eq!(
+    statement,
+    r#"SELECT `id` FROM `glyph` WHERE (TRUE) OR (FALSE)"#
+);
```
Comment (tyt2y3) [inline Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md:97]:
```suggestion
);

// a complex example
let (statement, values) = Query::select()
    .column(Glyph::Id)
    .from(Glyph::Table)
    .cond_where(
        Cond::all()
            .add(Cond::all().not())
            .add(Cond::any().not())
            .not(),
    )
    .build(MysqlQueryBuilder);

assert_eq!(
    statement,
    r#"SELECT `id` FROM `glyph` WHERE NOT ((NOT TRUE) AND (NOT FALSE))"#
);
```

File: Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    .from_function(Func::random(), Alias::new("func"))
+    .to_owned();
+
+assert_eq!(
+    query.to_string(MysqlQueryBuilder),
+    r#"SELECT * FROM RAND() AS `func`"#
+);
+```
+
+* [[#486](https://github.com/SeaQL/sea-query/pull/486)] Added binary operators from the Postgres `pg_trgm` extension
```
Comment (tyt2y3) [inline Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md:156]:
```suggestion
* [[#486](https://github.com/SeaQL/sea-query/pull/486)] Added binary operators from the Postgres `pg_trgm` extension
```rust
use sea_query::extension::postgres::PgBinOper;

assert_eq!(
    Query::select()
        .expr(Expr::col(Font::Name).binary(PgBinOper::WordSimilarity, Expr::value("serif")))
        .from(Font::Table)
        .to_string(PostgresQueryBuilder),
    r#"SELECT "name" <% 'serif' FROM "font""#
);
```
```

File: Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+* [[#473](https://github.com/SeaQL/sea-query/pull/473)] Added `ILIKE` and `NOT ILIKE` operators
+* [[#510](https://github.com/SeaQL/sea-query/pull/510)] Added the `mul` and `div` methods for `SimpleExpr`
+* [[#513](https://github.com/SeaQL/sea-query/pull/513)] Added the `MATCH`, `->` and `->>` operators for SQLite
```
Comment (tyt2y3) [inline Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md:159]:
```suggestion
* [[#513](https://github.com/SeaQL/sea-query/pull/513)] Added the `MATCH`, `->` and `->>` operators for SQLite
```rust
use sea_query::extension::sqlite::SqliteBinOper;

assert_eq!(
    Query::select()
        .column(Char::Character)
        .from(Char::Table)
        .and_where(Expr::col(Char::Character).binary(SqliteBinOper::Match, Expr::val("test")))
        .build(SqliteQueryBuilder),
    (
        r#"SELECT "character" FROM "character" WHERE "character" MATCH ?"#.to_owned(),
        Values(vec!["test".into()])
    )
);
```
```

File: Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    Query::select()
+        .column(Char::Character)
+        .from(Char::Table)
+        .and_where(Expr::col(Char::Character).binary(SqliteBinOper::Match, Expr::val("test")))
+        .build(SqliteQueryBuilder),
+    (
+        r#"SELECT "character" FROM "character" WHERE "character" MATCH ?"#.to_owned(),
+        Values(vec!["test".into()])
+    )
+);
```
Comment (tyt2y3) [inline Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md:200]:
```suggestion
);
```
```

Comment (tyt2y3) [inline Blog/blog/2022-12-30-whats-new-in-seaquery-0.28.0.md:200]: Ah sad, I just realized I can't suggest ```

**Review by tyt2y3 — APPROVED** [review]
Thank you

**Pull request merged**